### PR TITLE
[NEW] Display a view in a chat room in case it has no chats 

### PR DESCRIPTION
**Pull request by aniketsingh03**
@RocketChat/android

Closes https://github.com/RocketChat/Rocket.Chat.Android/issues/896

Screenshots:

img_1
img_2

### CODE REVIEW

**Comment by aniketsingh03** [comment]
@leonardoaramaki can you provide a drawable and information regarding its size and color of the text for this?

**Comment by leonardoaramaki** [comment]
@aniketsingh03 The drawable is on the line to be done by the design team but it's not prioritized. We'll have to wait

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt [hunk 1]
```diff
@@ -216,6 +213,10 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
         }
     }
 
+    override fun showNoMessagesView() {
+        no_messages_view.visibility = View.VISIBLE
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt]: You can use the `no_messages_view.setVisible(true)` here.

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt [hunk 2]
```diff
@@ -230,6 +231,11 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
     override fun showInvalidFileMessage() = showMessage(getString(R.string.msg_invalid_file))
 
     override fun showNewMessage(message: List<BaseViewModel<*>>) {
+        no_messages_view.visibility = View.GONE
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt]: `no_messages_view.setVisible(false)`

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -70,6 +70,7 @@
     <string name="msg_new_password">Enter New Password</string>
     <string name="msg_confirm_password">Confirm New Password</string>
     <string name="msg_unread_messages">Unread messages</string>
+    <string name="msg_no_chats_available">There are no messages yet in this channel</string>
```
Comment (philipbrito) [inline app/src/main/res/values/strings.xml]: There are no messages yet in this channel -> There are no messages in this channel yet.

For Brazilian translation:
Não há mensagens neste canal ainda.

File: app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt [hunk 4]
```diff
@@ -190,13 +193,7 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
...
         activity?.apply {
             if (recycler_view.adapter == null) {
-                adapter = ChatRoomAdapter(chatRoomType, chatRoomName, presenter,
-                        reactionListener = this@ChatRoomFragment)
-                recycler_view.adapter = adapter
-                val linearLayoutManager = LinearLayoutManager(context, LinearLayoutManager.VERTICAL, true)
-                linearLayoutManager.stackFromEnd = true
-                recycler_view.layoutManager = linearLayoutManager
-                recycler_view.itemAnimator = DefaultItemAnimator()
+                initialiseRecyclerViewAdapter()
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt:203]: Great! 👍

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Comment by aniketsingh03** [comment]
@filipedelimabrito I have made the requested changes

**Comment by aniketsingh03** [comment]
@leonardoaramaki is the drawable ready?

**Comment by aniketsingh03** [comment]
@leonardoaramaki @rafaelks @filipedelimabrito I have resolved some fresh conflicts, I can add the drawable if its ready.

**Pull request closed without merging**